test/metabase/query_processor_test/expression_aggregations_test.clj

```diff
@@ -274,3 +274,34 @@
                  (mt/run-mbql-query venues
                    {:aggregation [[:sum [:expression "double-price"]]]
                     :expressions {"double-price" [:* $price 2]}}))))))))
+
+(deftest two-case-functions-test
+  (testing "We should support expressions with two case statements (#15107)"
+    (mt/test-drivers (mt/normal-drivers-with-feature :expressions)
+      (mt/dataset sample-dataset
+        (is (= [[1
+                 "1018947080336"
+                 "Rustic Paper Wallet"
+                 "Gizmo"
+                 "Swaniawski, Casper and Hilll"
+                 29.46
+                 4.6
+                 "2017-07-19T19:44:56.582Z"
+                 1]
+                [2
+                 "7663515285824"
+                 "Small Marble Shoes"
+                 "Doohickey"
+                 "Balistreri-Ankunding"
+                 70.08
+                 0.0
+                 "2019-04-11T08:49:35.932Z"
```

Comment:
Looks like this cause a test failure with [sqlite-ee](https://app.circleci.com/pipelines/github/metabase/metabase/13841/workflows/0f915834-bc89-4d70-8d4c-370428e0f8d5/jobs/529801).

Not sure how, but the actual result is (no extra digits in the seconds):
```
"2019-04-11T08:49:35Z"
```

Reply:
fixed